lib/index.js

```diff
@@ -3,9 +3,15 @@
 import LinterUI from './main'
 import type Intentions from './intentions'
 
-const idleCallbacks = new Set()
+const idleCallbacks: Set<any> = new Set()
```

Comment:
Could we make this `Set<number>` please?

Reply:
[The spec](https://developer.mozilla.org/en-US/docs/Web/API/Window/requestIdleCallback) only specifies "identifier", it doesn't actually say what that identifier _is_, so `<any>` is appropriate here.